src/main/java/cz/czechitas/automation/ApplicationAction.java

```diff
@@ -18,42 +18,42 @@ final class ApplicationAction {
         this.elementFinder = Objects.requireNonNull(elementFinder);
     }
 
+    /**
+     * Click Vytvorit novou prihlasku on table view
+     */
     void clickCreateNewApplicationButton() {
-        var createApplicationButton = elementFinder.findByXPath("/html/body/div/div/div/div/div/div[1]/a");
+        var createApplicationButton = elementFinder.findByCssSelector(".card-header a");
         createApplicationButton.click();
     }
 
     void selectProgrammingSection() {
-        var programmingSectionButton = elementFinder.findByXPath("/html/body/div/div/div[1]/div[3]/div/div[2]/a");
+        var programmingSectionButton = elementFinder.findByXPath("//*[contains(text(), 'Programování')]/../..//a");
```

Comment:
```suggestion
        var programmingSectionButton = elementFinder.findByXPath("//*[contains(text(), 'Programování')]//ancestor::*[@class='card']//a");
```

My personal preference is to not rely on absolute paths to ancestors 🙂